source/ways-of-working/guides/environments.html.md.erb

```diff
@@ -0,0 +1,22 @@
+---
+title: Environments
+last_reviewed_on: 2021-11-01
+review_in: 6 months
+weight: 10
+---
+
+# <%= current_page.data.title %>
+
+
+
+| Shared Services Environment | CNP Environment | Use case &nbsp; | Path to live | Additional Information &nbsp; &nbsp; |
+| - | - | - | - | - | - |
+| Prod            | Prod       | Live services | Yes |  |
+| staging         | AAT        | Automated Acceptance Testing before moving to Prod | Yes |  |
+| Demo            | Demo       | External demonstrations (no VPN required) | No | Used for demonstrating to external stakeholders |
```

Comment:
could you make 'Used for demonstrating to external stakeholders' a list and move no vpn required to the additional information as well